app/controllers/account_activations_controller.rb

```diff
@@ -1,6 +1,7 @@
 class AccountActivationsController < ApplicationController
   def edit
     user = User.find_by email: params[:email]
+    # binding.pry
```

Comment:
.

Reply:
tách hẵng code này ra 1 line về find user đi
handle khi user not found luôn và dùng action filter để goi